meetings/update/update_meeting.js

```diff
@@ -0,0 +1,129 @@
+/**
+ *                _               
+ *  __      _____| |__   _____  __
+ *  \ \ /\ / / _ \ '_ \ / _ \ \/ /
+ *   \ V  V /  __/ |_) |  __/>  <         @WebexDevs
+ *    \_/\_/ \___|_.__/ \___/_/\_\
+ * 
+ * UPDATE a meeting in Webex with the REST API in Node
+ * https://developer.webex.com/docs/api/v1/meetings/create-a-meeting
+ * 
+ * Step 0: Have a (free) Webex account: https://cart.webex.com/sign-up
+ * Step 1: Log in to https://developer.webex.com/login
+ * Step 2: Find your bearer token at
+ *         https://developer.webex.com/docs/getting-started under "Your
+ *         Personal Access Token" in the middle of the page.
+ * Step 3: Replace the string on the line that defines const myWebexDeveloperToken,
+ *         just below, with your personal bearer (access) token. Hit "save".
+ * Step 4: Run an example from https://github.com/WebexSamples/rest-api-samples/tree/main/meetings/read
+ *         to obtain a meeting ID. Replace line that defines path in options object with your meeting ID.
+ * Step 5: Run this file with node from within
+ *         this directory on the command line:
+ * 
+ *         node ./update_meeting.js
+ * 
+ * Step 6: Profit. Get your app listed in the Webex App Hub!
+ *         https://apphub.webex.com/
+ * 
+ */
+
+ const https = require('https'); // https://nodejs.org/api/https.html
+
+
+const myWebexDeveloperToken = 'REPLACE WITH API KEY';
+
+const body = JSON.stringify({
+   title: 'Incorporating Vibranium into Modern Electronics: Pt. 1', // String, Required | Meeting title. The title can be a maximum of 128 characters long.
+   agenda: 'This meeting`s agenda includes discussing plans to incorporate new, extremely conductive copper alloys which create their own electro-magnetic fields.', // example of one of many options to update
+   password: 'REPLACE WITH MEETING PASSWORD',                  // String, Required
```

Comment:
Let's move the definition of this up top next to the `myWebexDeveloperToken` const.